packages/examples/package.json

```diff
@@ -17,7 +17,18 @@
         "build": "tsc -b",
         "lint": "eslint .",
         "lint:fix": "eslint . --fix",
-        "format": "prettier --write \"src/**/*.ts\" \"**/*.json\""
+        "format": "prettier --write \"src/**/*.ts\" \"**/*.json\"",
+        "demo:coap:server": "node dist/bindings/coap/example-server.js",
+        "demo:coap:client": "node dist/bindings/coap/example-client.js",
+        "demo:http:server": "node dist/bindings/http/example-server.js",
+        "demo:http:server-secure": "node dist/bindings/http/example-server-secure.js",
+        "demo:http:client": "node dist/bindings/http/example-client.js",
+        "demo:opcua:1": "node dist/bindings/opcua/opcua-demo1.js",
+        "demo:opcua:2": "node dist/bindings/opcua/opcua-demo2.js",
+        "demo:opcua:coffee-machine": "node dist/bindings/opcua/opcua-coffee-machine-demo.js",
+        "quickstart:smart-clock": "node dist/quickstart/smart-clock.js",
+        "quickstart:simple-coffee-machine": "node dist/quickstart/simple-coffee-machine.js",
+        "quickstart:presence-sensor": "node dist/quickstart/presence-sensor.js"
```

Comment:
I wonder "what" are the best names and locations ?

Personally, I would not use "demo" and use "example" instead since we are in the `examples` package.
OR, since we are already in examples package the folder they are located in

"demo:coap:server": "node dist/bindings/coap/example-server.js",
-->
"binding:coap:server": "node dist/bindings/coap/example-server.js",

Matches with quickstart also.

What do others think?